packages/react/src/section-alert/SectionAlert.tsx

```diff
@@ -58,34 +59,40 @@ export const SectionAlert = forwardRef<HTMLDivElement, SectionAlertProps>(
 			focusOnUpdate,
 			forwardedRef,
 		});
+		const { childrenId, iconId, titleId } = useSectionAlertIds(id);
 
 		const Icon = sectionAlertIconMap[tone];
 		const closeHandler = getOptionalCloseHandler(onClose, onDismiss);
 
 		return (
 			<Flex
+				{...props}
 				alignItems="center"
+				aria-labelledby={`${iconId} ${titleId} ${children ? childrenId : ''}`}
 				background={tone}
 				borderColor={tone}
 				borderLeft
 				borderLeftWidth="xl"
+				focusRingFor="all"
 				gap={0.5}
 				highContrastOutline
 				id={id}
-				focusRingFor="all"
 				justifyContent="space-between"
 				padding={1}
 				ref={ref}
-				role={role}
+				role={role || 'region'}
```

Comment:
Didn't use a default arg because if someone accidentally passes `false`, I still need `region` to be set

Reply:
That's likely worth a code comment so that some dev doesn't come back after the holidays and think they can "improve" the code. Some dev, aka me.

Reply:
Haha I thought the same when I added this comment

Reply:
Done